starfish/image/_segmentation/watershed.py

```diff
@@ -321,6 +321,28 @@ def watershed(self, markers: np.ndarray, watershed_mask: np.ndarray) -> np.ndarr
 
         return res
 
+    @staticmethod
+    def relabel_image(image: np.ndarray) -> np.ndarray:
+        """given a label image where some objects have been removed, relabel it with sequential integers
+
+        Parameters
+        ----------
+        image : np.ndarray[np.uint32]
+            image whose values identify which object each pixel corresponds to. the values may
+            not be sequential integers.
+
+        Returns
+        -------
+        image : np.ndarray[np.uint32]
+            same as input, but the values are re-labled as sequential integers
+        num_labels : int
+            number of unique objects
+        """
+        output = np.zeros_like(image)
```

Comment:
empty_like?

Reply:
The "not an object" default for each pixel is zero, so `np.zeros` is the correct method for this.